vnext/Microsoft.ReactNative/Microsoft.ReactNative.vcxproj.filters

```diff
@@ -730,6 +732,7 @@
     <Midl Include="Views\cppwinrt\ViewPanel.idl">
       <Filter>Views\cppwinrt</Filter>
     </Midl>
+    <Midl Include="JsiApi.idl" />
```

Comment:
><Midl Include="JsiApi.idl" /> [](start = 4, length = 29)

[nit] Associate with a filter?

Reply:
We can do it in future. All current IDL files are not associated with a filter yet.

---
In reply to: [451852274](https://github.com/microsoft/react-native-windows/pull/5461#discussion_r451852274) [](ancestors = 451852274)